rudder-agent/SOURCES/rudder-agent-postinst

```diff
@@ -73,6 +73,8 @@ if [ ! -f /var/rudder/cfengine-community/inputs/promises.cf ] || ! /var/rudder/c
 then
   rm -rf /var/rudder/cfengine-community/inputs/* || true
   cp -r /opt/rudder/share/initial-promises/* /var/rudder/cfengine-community/inputs/
+  rm -rf /var/rudder/ncf/common/*
+  cp -r /opt/rudder/share/initial-ncf/* /var/rudder/ncf/common/
 fi
```

Comment:
In the case of a "normal" start (agent not disabled + not first install) there is no specific install message, so that's fine by me. We do need to keep the install specific messages when agent is disabled or not yet configured, here, though.